library/index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <main class="index-page">
+        <!-- header section -->
+        <header>
+            <nav class="nav">
+                <h3 class="logo">OurLibrary</h3>
+                <ul>
+                    <a href="#">
+                        <li>Home</li>
+                    </a>
+                    <li id="appear-about">About</li>
+                </ul>
+            </nav>
+        </header>
+        <!-- header section end -->
+        <!-- about section -->
+        <section id="about" class="about">
+            <button id="closeBtn">&times;</button>
+            <article class="about-content">
+                <h3>THE IMPORTANCE OF READING IN OUR LIFE</h3>
+                <p>Reading is important for many reasons. Firstly, it develops our thoughts. Secondly, it gives us
+                    endless knowledge and lessons. Thirdly, it keeps our minds active. Finally, books can hold and keep
+                    all kinds of information, stories, thoughts and feelings unlike anything else.
+                </p><br>
+                <p id="sec-p">So we have built this website to give you the opportunity to discover an intersting world.
+                </p>
+            </article>
+        </section>
+        <!-- about section end -->
+        <!-- Start the Laibrary section -->
+        <section class="Library">
+            <h1>Library</h1>
+            <p>This Is Our Library We Hope You<br>
+                Find The Books You Look For<br>
+                ‏If You Like More Books <br>
+                We Invite You to Serch Here</p>
+            <button>Go Search</button>
+        </section>
+        <!-- End the Laibrary section -->
+        </div>
+        <!-- Start image section -->
+        <section class="img-section">
+
+            <div id="images">
+                <img src="../images/img_library/pic1.jpg" alt="Twilight">
```

Comment:
**use the semantic element `figure` with an image here and do the same with the rest .**